x-pack/plugins/cloud_security_posture/public/pages/findings/findings_by_resource_table.tsx

```diff
@@ -0,0 +1,111 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import React from 'react';
+import {
+  EuiTableFieldDataColumnType,
+  EuiEmptyPrompt,
+  EuiBasicTable,
+  EuiTextColor,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiLink,
+  EuiToolTip,
+} from '@elastic/eui';
+import { FormattedMessage } from '@kbn/i18n-react';
+import numeral from '@elastic/numeral';
+import { extractErrorMessage } from '../../../common/utils/helpers';
+import * as TEST_SUBJECTS from './test_subjects';
+import * as TEXT from './translations';
+import type { CspFindingsByResourceResult } from './use_findings_by_resource';
+
+const formatNumber = (value: number) => (value < 1000 ? value : numeral(value).format('0.0a'));
```

Comment:
this needs to be moved to `common` 
only used because `FormattedNumber` doesn't support `compact` mode yet

Reply:
😢 
Did you figure out why `FormattedNumber` does not support `notation="compact"`? Is it due to `react-intl` version or something? If you find out, please add a TODO and link to the relevant Kibana ticket

Reply:
IIRC we're not using the native browser `Intl` in current version. not sure why tbh, seems like if it exists it should always fallback to it.